display_information/control-annotation-sublayer-visibility/src/main/java/com/esri/samples/control_annotation_sublayer_visibility/ControlAnnotationSublayerVisibilitySample.java

```diff
@@ -81,9 +81,11 @@ public void start(Stage stage) {
 
       // show current map scale in a label within the control panel
       Label currentMapScaleLabel = new Label();
-      mapView.addMapScaleChangedListener(mapScaleChangedEvent ->
-              currentMapScaleLabel.setText("Scale: 1:" + Math.round(mapView.getMapScale()))
-      );
+
+      // listen for map scale changes and update the label
+      mapView.mapScaleProperty().addListener((observable, oldValue, newValue) -> {
+        currentMapScaleLabel.setText("Scale: 1:" + Math.round((double) newValue));
+      });
```

Comment:
```suggestion
      mapView.mapScaleProperty().addListener((observable, oldValue, newValue) -> 
        currentMapScaleLabel.setText("Scale: 1:" + Math.round((double) newValue)));
```
Change looks good, it can just be switched to a lambda (without the brackets)